guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+## Table of contents
+
+1. [How we write](#how-we-write)
+2. [Grammar and punctuation](#grammar-and-punctuation)
+3. [Links](#links)
+4. [Lists](#lists)
+5. [Numbers](#numbers)
+6. [Symbols, currency and abbreviations](#Symbols,-currency-and-abbreviations)
+7. [Common terms](#common-terms)
+
+
+## How we write
+
+#### Voice and tone
+
+MetaBrainz speaks in a friendly, direct and easily understandable manner. We:
+* use plain, simple, language when possible
+* use short sentences
+* use an active voice (avoid the passive tense).
+
+#### Spelling
+
+We use US English.
+
+#### Active and passive writing 
+
+Active writing mentions the subject (the person or thing 'doing' the action) first in the sentence (for example, inflation pushed up house prices).
+
+Passive writing mentions the object (the person or thing 'receiving' the action) first. Passive sentences often include 'by' (for example, house prices were pushed up by inflation).
+
+Active writing is more direct and should be used most of the time. We use short, clear sentences.
+
+
+## Grammar and punctuation
+
+#### Exclamation marks
+
+We don't use exclamation marks in most instances - for instance, we don't use exclamation marks in documentation or page content.
+
+#### Colon and semi-colons
+
+We avoid these. When possible we’ll write 2 sentences instead, or separate the clauses using a dash (with a space on either side).
+
+#### Apostrophes
+
+We generally don’t add an extra 's' after nouns or names ending in 's'.
+For example: "The business’ work" – not  "the business’s work."
+
+We don’t use an apostrophe for dates, numbers, or plurals of abbreviations.
+For example: 1960s.
+
+#### Bold
+
+We rarely use bold – using too much will make it difficult for users to know which parts of your content they need to pay the most attention to. To emphasise words or phrases, we:
+* put key information at the start of sentences
+* use headings
+
+#### Capitals
+
+We only use capitals for proper nouns, such as:
+* names of people, places and things, including buildings and brands
+* titles, such as: of books, albums, recordings
+
+'Types' of places are not capitalized, unless referring to a specific place.
+For example: "I like the library" - and "I like my local Library"
+
+Generally, terms are not proper nouns, so should not be capitalised. Technical terms are not proper nouns. But if a word or term is branded as a distinct thing, treat it as a proper noun.
+
+#### Headings
+
+We use sentence case so only the first letter is upper case. We never link headings.
+
+#### Quotation marks
+
+We use double quotation marks for:
+* exact quotations
+* direct speech.
+
+We use single quotation marks for:
+* technical terms (the first time it is used)
+* classification descriptors
+* titles of documents or publications
```

Comment:
Why not just always use double quotation marks?

Reply:
If you write "technical term" with double guotes I assume you're quoting a line from someone else or from a specific source, which wouldn't be the case with these. Perhaps that's just what I learnt in school? But that makes sense to me

Reply:
Sure, I guess my idea was "drop those two, use double quotes for titles". But if single quotes for that is a generally accepted convention, then fine by me.